unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneController.cs

```diff
@@ -191,6 +193,10 @@ private void SortScenesByDistance()
                     if (scene.sceneData.id != globalSceneId && characterIsInsideScene)
                     {
                         currentSceneId = scene.sceneData.id;
+
+                        if (firstSort)
```

Comment:
Not a fan of this approach because suddenly, the scenes sorting method has the loading lock responsibility. Ideally i'd think of a way to keep this responsibility outside.